app.js

```diff
@@ -11,6 +11,8 @@ const favicon = require("serve-favicon");
 const errorhandler = require("errorhandler");
 const nunjucks = require("nunjucks");
 
+const mousewheel = require("jquery-mousewheel");
```

Comment:
super minor, but can you remove the empty L13?

Reply:
fixed!